pulp_ansible/app/tasks/collections.py

```diff
@@ -455,6 +476,9 @@ async def _add_collection_version(self, api_version, collection_version_url, met
             version=metadata["version"],
         )
         cv_unique = attrgetter("namespace", "name", "version")(collection_version)
+        fullname = ".".join(cv_unique[:2])
+        if fullname in self.exclude_info and cv_unique[2] in self.exclude_info[fullname]:
```

Comment:
could you add a test for checking the scenario when we want to exclude only a specific version?

Reply:
```suggestion
        fullname = ".".join(cv_unique[:2])
        if fullname in self.exclude_info and collection_version.version in self.exclude_info[fullname]:
```
`cv_unique[2]` made me confuse, I had to read multiple times to understand